### PR TITLE
Performance graph feature

### LINKED ISSUE #104: Implement Performance Tracking
**As a user, I can view the performance history of a employee, so that I can monitor overall performance of employee.**

Comprises of:

- Adding performance as a field to the Person objects #97 
- Enabling viewing of the performance field as a graph #101

### PR DESCRIPTION
**Pull request by Kok-je**
Completed #101 
Resolves #104 

**Brief Overview**:
- Implement a graphical visualisation of employee past ratings with a timestamp
- JavaFX `lineChart` displaying person `performanceHistory` array values of `rating` and `timestamp`
- Reordered Person Info data layout [UI]
- Designed and updated Coydir Logo to reflect in `MainWindowFXML`

### CODE REVIEW

**Review by chao890 — APPROVED** [review]
LGTM

File: src/main/resources/view/PersonInfo.fxml [hunk 1]
```diff
@@ -32,10 +35,24 @@
             <Label fx:id="email" styleClass="info_small_label" text="\$email" />
             <Label fx:id="phone" styleClass="info_small_label" text="\$phone" />
             <Label fx:id="address" styleClass="info_small_label" text="\$address" />
+            <Label fx:id="performance" styleClass="info_small_label" text="\$performance" />
+            <HBox>
+                <padding>
+                    <Insets top="10"/>
+                </padding>
+                <LineChart fx:id="lineChart" layoutX="14.0" layoutY="52.0" prefHeight="512.0"
+                    prefWidth="887.0" title="Fuck" style="-fx-background-color: #F5F5F5;">
```
Comment (chao890) [inline src/main/resources/view/PersonInfo.fxml:44]: Maybe use other word for title?

**Review by chao890 — CHANGES_REQUESTED** [review]

File: src/main/resources/view/PersonInfo.fxml (hunk 1 above)
Comment (prit3010) [reply src/main/resources/view/PersonInfo.fxml:44]: I agree. Does not seem appropriate

**Review by prit3010 — CHANGES_REQUESTED** [review]

**Review by prit3010 — APPROVED** [review]
Looks better

**Review by chao890 — APPROVED** [review]
LGTM

**Pull request merged**